public/components/model_version/model_version.tsx

```diff
@@ -73,9 +73,12 @@ export const ModelVersion = () => {
           { key: 'Precision', value: '0.64', type: 'number' as const },
           { key: 'Task', value: 'Image classification', type: 'string' as const },
         ], // TODO: read from model.tags
-        configuration: JSON.stringify(model.model_config),
+        configuration: JSON.stringify(model.model_config, undefined, 2),
         modelFileFormat: model.model_format,
         // TODO: read model url or model filename
+        artifactSource: 'source_not_changed',
+        // modelFile: new File([], 'artifact.zip'),
```

Comment:
Do we need to construct the model file in the API integration phase?

Reply:
I'm thinking to create an empty File instance with the model file name we get from API. What do you think?

Reply:
I think that's make sense to init an empty file.